packages/phone-number-privacy/combiner/src/common/io.ts

```diff
@@ -62,9 +62,11 @@ export abstract class IO<R extends OdisRequest> {
   }
 
   getKeyVersionInfo(request: Request<{}, {}, OdisRequest>, logger: Logger): KeyVersionInfo {
-    const requestKeyVersion = getRequestKeyVersion(request, logger)
-    const defaultKeyVersion = this.config.keys.currentVersion
-    const keyVersion = requestKeyVersion ?? defaultKeyVersion
+    let requestKeyVersion: number | undefined
```

Comment:
can you add a comment explaining that we don't want this to throw an error if an invalid key version is passed in?